chart/values.yaml

```diff
@@ -2030,6 +2030,7 @@ pgbouncer:
         command: ["/bin/sh", "-c", "killall -INT pgbouncer && sleep 120"]
 
   metricsExporterSidecar:
+    enabled: false
```

Comment:
Can we set the default to true? It is a change in behaviour and might break backward compatibility

Reply:
Definitely, default value should be true to keep it b/c.